tests/e2e/tests/pilot/authn_policy_test.go

```diff
@@ -67,3 +68,76 @@ func TestAuthNPolicy(t *testing.T) {
 		}
 	})
 }
+
+func TestAuthNJwt(t *testing.T) {
+	// V1alpha3 == true implies envoyv2, jwt authn doesn't work for v2 so skip it now.
+	if tc.V1alpha3 {
+		t.Skipf("Skipping %s: V1alpha3=true", t.Name())
+	}
+
+	// JWT token used is borrowed from https://github.com/istio/proxy/blob/master/src/envoy/http/jwt_auth/sample/correct_jwt.
```

Comment:
Does that jwt depend on the [public key](https://github.com/istio/proxy/blob/master/src/envoy/http/jwt_auth/sample/pubkey.jwk), but in this test, you use actual google service account. Could they be out of sync?

Reply:
Since issuer is a google service account, I think it should be fine to use latest key from jwks_uri(format like https://www.googleapis.com/service_accounts/v1/jwk/<YOUR-SVC-ACCOUNT>)